content/sigs/ux/index.adoc

```diff
@@ -32,13 +32,13 @@ An overview of the Jenkins UX SIG documentation can be found by following this l
 * Discussing UI frameworks
 * Discussing impact of UI changes to existing plugins
 
-=== Slack Channel
-The UX SIG uses Slack as the primary way to discuss any relevant topics. 
-The channel used for this is #jenkins-ux-sig on the CDF Slack group. 
-You can join the Slack channel via this link:https://join.slack.com/t/cdeliveryfdn/shared_invite/enQtODM2NDI1NDc0MzIxLTA1MDcxMzUyMGU2NWVlNmQwN2M1N2M4MWJjOWFkM2UzMDY0OWNkNjAzNzM0NzVkNjQ5M2NkMmY2MTRkMWY4MWY[link].
+=== Gitter Channel
+The UX SIG use Gitter as the primary way to discuss any relevant topics. Initially this used to be a CDF Slack channel but a decision was made to move to Gitter in early February 2020. 
```

Comment:
Typically we've preferred sentence per line for easier visualization of changes from one version to the next.
Not required, just for your consideration.
```suggestion
The UX SIG uses Gitter to discuss relevant topics.
Initially this used to be a CDF Slack channel but a decision was made to move to Gitter in early February 2020. 
```